install/kubernetes/tetragon/values.yaml

```diff
@@ -247,6 +247,8 @@ tetragon:
   # pod association. This feature depends on cri being enabled.
   cgidmap:
     enabled: false
+  # -- Read-only mount /sys/kernel/security into Tetragon container to check if LSM BPF is enabled on the host.
+  enableLsmSensor: false
```

Comment:
Can we do
```
  lsm:
    enabled: false
```
instead? That's more conventional in Helm, and if we need more LSM sensor-related values in the future, they'll be nicely grouped.